app/pb_kits/playbook/pb_popover/docs/_popover_default.md

```diff
@@ -0,0 +1,11 @@
+# Details
+-Popovers rely on the 3rd party library Popper.js for positioning. You must include popper.min.js before bootstrap.js or use bootstrap.bundle.min.js / bootstrap.bundle.js which contains 
+-Popper.js in order for popovers to work!
```

Comment:
Is this true? Doesn't this come from vendor.js? Why do mention bootstrap

Reply:
Can we hold off on this documentation? I want to discuss it

Reply:
This was dummy data. @jasoncypret Removing this now.